runtime/syntax/testdir/input/codeowners.txt

```diff
@@ -0,0 +1,69 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
```

Comment:
This is GitHub's CODEOWNERS reference example file. I can reduce it by quite a lot if we'd prefer it to be simpler.